cashctrl_ledger/ledger.py

```diff
@@ -281,6 +365,25 @@ def delete_account(self, account: str, allow_missing: bool = False):
             self._client.post("account/delete.json", {"ids": delete_id})
             self._client.invalidate_accounts_cache()
 
+    def delete_accounts(self, accounts: List[int] = [], allow_missing: bool = False):
+        """Deletes accounts from the remote CashCtrl instance.
+
+        Args:
+            accounts (str[]): The account numbers to be deleted.
+            allow_missing (bool, optional): If True, do not raise an error if the
+                                            account is missing. Defaults to False.
+        """
+        if isinstance(accounts, int):
+            accounts = [accounts]
```

Comment:
I believe that only checking for int, but not for any other singleton type is too narrow. Account number could also be a floats or a strings.

Please pick one of two proposed solutions:
1. Drop L376:377, meaning that we always expect a list as arguments.
1. Research how to properly check for singletons in Python